std_msgs/Header header
string camera_name
string vehicle_name
geometry_msgs/Quaternion orientation